knative-operator/pkg/controller/dashboard/testdata/grafana-dash-knative-eventing-channel.yaml

```diff
@@ -0,0 +1,1065 @@
+                "format": "time_series",
+                "legendFormat": "p95",
+                "refId": "C"
+              },
+              {
+                "expr": "histogram_quantile(0.99, sum(rate(kafkachannel_dispatcher_event_dispatch_latencies_bucket{namespace_name=~\"$eventNamespace\"}[1m])) by (le))",
+                "format": "time_series",
+                "legendFormat": "p99",
+                "refId": "D"
+              }
+            ],
+            "thresholds": [],
+            "timeFrom": null,
+            "timeRegions": [],
+            "timeShift": null,
+            "title": "Kafka Dispatcher: Event Dispatch Latency (ms)",
+            "tooltip": {
+              "shared": true,
+              "sort": 0,
+              "value_type": "individual"
+            },
+            "type": "graph",
+            "xaxis": {
+              "buckets": null,
+              "mode": "time",
+              "name": null,
+              "show": true,
+              "values": []
+            },
+            "yaxes": [
+              {
+                "decimals": 2,
+                "format": "ms",
+                "label": null,
+                "logBase": 1,
+                "max": null,
+                "min": null,
+                "show": true
+              },
+              {
+                "format": "short",
+                "label": null,
+                "logBase": 1,
+                "max": null,
+                "min": null,
+                "show": true
+              }
+            ],
+            "yaxis": {
+              "align": false,
+              "alignLevel": null
+            }
+          }
+      ],
+      "templating": {
+        "list": [
+          {
+            "allValue": null,
+            "current": {},
+            "datasource": "prometheus",
+            "hide": 0,
+            "includeAll": false,
+            "label": "Namespace",
+            "multi": false,
+            "name": "eventNamespace",
+            "options": [],
+            "query": "label_values((kafkachannel_dispatcher_event_count{} OR inmemorychannel_dispatcher_event_count{}), namespace_name)",
+            "refresh": 2,
+            "regex": "",
+            "sort": 1,
+            "tagValuesQuery": "",
+            "tags": [],
+            "tagsQuery": "",
+            "type": "query",
+            "useTags": false
+          }
+        ]
+      },
+      "refresh": false,
+      "schemaVersion": 19,
+      "style": "dark",
+      "tags": [],
+      "time": {
+        "from": "now-6h",
+        "to": "now"
+      },
+      "timepicker": {
+        "refresh_intervals": [
+          "5s",
+          "10s",
+          "30s",
+          "1m",
+          "5m",
+          "15m",
+          "30m",
+          "1h",
+          "2h",
+          "1d"
+        ]
+      },
+      "timezone": "",
+      "title": "Knative Eventing - Channel",
```

Comment:
since these has IMC and KC channels.  Should the title say `Channels` ?

Reply:
I didnt use plural for the other dashboards so I kept this convention here too.